packages/unified-selection/src/test/_helpers/Utils.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { BeDuration, StopWatch } from "@itwin/core-bentley";
+
+export async function waitFor<T>(check: () => Promise<T> | T, timeout?: number): Promise<T> {
```

Comment:
use https://github.com/iTwin/presentation/blob/1a0cd35be662e7ccead170a2492e10d35829b371/packages/test-utilities/src/test-utilities/WaitFor.ts#L8 instead

Reply:
Switched to using `presentation-test-utilities`.